pkg/debuggers/remote/utils.go

```diff
@@ -37,13 +37,21 @@ func GetPort(pid int) (int, error) {
 	return port, nil
 }
 
-func GetPortOfJavaProcess(pid int) (int, error) {
+func GetPortOfJavaProcess(pid int, env map[string]string) (int, error) {
 
 	args, err := utils.GetCmdArgsByPid(pid)
 	if err != nil {
 		log.WithFields(log.Fields{"pid": pid, "err": err}).Error("Can't get command line arguments")
 		return 0, err
 	}
+	s, ok := env["JAVA_TOOL_OPTIONS"]
+	if ok {
+		s = strings.Replace(s, "\x00", " ", -1)
```

Comment:
as below for the command line argument, it would be helpful to put some comments explaining what sort of strings to expect as the environment variable value

Reply:
Its basically from here https://github.com/solo-io/squash/blob/e42715ca201a662c7e09d0e9f44ea4061284c261/pkg/utils/utils.go#L33. I thought its a good practice to remove NULL at end of line and following the same :)

Reply:
that's fine, worth dropping a comment as the intent is non obvious.
Regarding examples, it'd be nice to cite what a user should expect the `JAVA_TOOL_OPTIONS` value to contain. Like this does for the cli: https://github.com/solo-io/squash/blob/ccc91d8242928ee213ae3317012c33c3f0e1bb39/pkg/debuggers/remote/utils.go#L57-L59